src/main/js/main_html5.js

```diff
@@ -100,10 +101,18 @@ require("../../../html5-common/js/utils/environment.js");
 
       video.css(css);
 
-      // enable airplay for iOS
-      // http://developer.apple.com/library/safari/#documentation/AudioVideo/Conceptual/AirPlayGuide/OptingInorOutofAirPlay/OptingInorOutofAirPlay.html
       if (OO.isIos) {
+        // enable airplay for iOS
+        // http://developer.apple.com/library/safari/#documentation/AudioVideo/Conceptual/AirPlayGuide/OptingInorOutofAirPlay/OptingInorOutofAirPlay.html
+        //
         video.attr("x-webkit-airplay", "allow");
+
+        //enable inline playback for mobile
+        if (pluginParams["mobileInlinePlayback"] === true) {
+          if (OO.iosMajorVersion >= 10) {
+            video.attr('playsinline', '');
```

Comment:
Are there other possible values for this attribute? I couldn't easily find documentation about possible values online.